tools/Silverback.Tools.KafkaConfigClassGenerator/ProxyClassGenerator.cs

```diff
@@ -137,8 +137,14 @@ private void MapProperties()
                 _builder.AppendLine($"        public {propertyType} {property.Name}");
                 _builder.AppendLine("        {");
 
-                if (property.GetGetMethod() != null)
+                if (property.Name == "GroupId" && property.GetGetMethod() != null)
```

Comment:
The `GroupId` should be `abstract` as well. (Generating the proxy class with the tool and replacing the code you manually adjusted leads to non-compiling code.)

Reply:
Thanks for noticing, I has missed it!
Fixed: https://github.com/BEagle1984/silverback/pull/172/commits/62e9b8b415772dfaab2ac3f648be3fb709e33cde